receiver/mysqlreceiver/client.go

```diff
@@ -73,6 +97,63 @@ func (c *mySQLClient) getInnodbStats() (map[string]string, error) {
 	return Query(*c, query)
 }
 
+// getTableIoWaitsStats queries the db for table_io_waits metrics.
+func (c *mySQLClient) getTableIoWaitsStats() ([]TableIoWaitsStats, error) {
+	query := `
+	SELECT OBJECT_SCHEMA, OBJECT_NAME, COUNT_DELETE, COUNT_FETCH, COUNT_INSERT, COUNT_UPDATE,
+	SUM_TIMER_DELETE, SUM_TIMER_FETCH, SUM_TIMER_INSERT, SUM_TIMER_UPDATE
+	FROM performance_schema.table_io_waits_summary_by_table
+	WHERE OBJECT_SCHEMA NOT IN ('mysql', 'performance_schema');
+`
```

Comment:
Presumably the extra whitespace is ignored by the database, but is it worth building the string in such a way where it is nicely formatted. I imagine users who see the query in the database's query log may find this format unusual.

Reply:
I changed it a bit:
```
SELECT OBJECT_SCHEMA, OBJECT_NAME, COUNT_DELETE, COUNT_FETCH, COUNT_INSERT, COUNT_UPDATE,
	SUM_TIMER_DELETE, SUM_TIMER_FETCH, SUM_TIMER_INSERT, SUM_TIMER_UPDATE
FROM performance_schema.table_io_waits_summary_by_table
WHERE OBJECT_SCHEMA NOT IN ('mysql', 'performance_schema');
```

Reply:
This still seems odd to me. It reads nicely in code, but why should it contain return chars and tabs that are useful only for the golang code? How about something like this:
```go
query := "SELECT OBJECT_SCHEMA, OBJECT_NAME, COUNT_DELETE, COUNT_FETCH, COUNT_INSERT, " +
	"COUNT_UPDATE, SUM_TIMER_DELETE, SUM_TIMER_FETCH, SUM_TIMER_INSERT, SUM_TIMER_UPDATE " +
	"FROM performance_schema.table_io_waits_summary_by_table " +
	"WHERE OBJECT_SCHEMA NOT IN ('mysql', 'performance_schema');"
```
It's simple and reads clearly in both go and sql domains.